README.rst

```diff
@@ -178,6 +178,11 @@ In this case, extra *docker run* options can be provided in your *"registry:look
 
 By default, the storage backend used by the registry is "filesystem". Use environment variables to override that, for example to use S3 as backend storage.
 
+Development
+===========
+
+Note that some of the internal states such as `docker.running` are references to the internal `dockerio states <https://docs.saltstack.com/en/latest/ref/states/all/salt.states.dockerio.html>`
```

Comment:
You are correct. Thank you for pointing me in the right direction. I was looking for `dockerio`. I've only used `dockerng` from the SaltStack API. In addition, I think your double usage of `internal` confused me. Not a problem.

This would probably be better in the only file that uses those states. What do you think? Putting it there would likely give need to change the wording.